scripts/security/certs-create.sh

```diff
@@ -23,7 +23,8 @@ do
 				 -keystore kafka.$i.keystore.jks \
 				 -keyalg RSA \
 				 -storepass confluent \
-				 -keypass confluent
+				 -keypass confluent \
+				 -deststoretype pkcs12
```

Comment:
I do not see option "-deststoretype" for -genkeypair command. Is it supposed to use "-storetype"?

Reply:
Good catch @jeffhuang26 .  I wonder how this accidentally works.

@rspurgeon FWIW I tested with `storetype` instead of `deststoretype` and it appears to work as well:

```
cp-demo(DEVX-1032) ✗: git diff
diff --git a/scripts/security/certs-create.sh b/scripts/security/certs-create.sh
index e6e7381..bc6318b 100755
--- a/scripts/security/certs-create.sh
+++ b/scripts/security/certs-create.sh
@@ -24,7 +24,7 @@ do
                                 -keyalg RSA \
                                 -storepass confluent \
                                 -keypass confluent \
-                                -deststoretype pkcs12
+                                -storetype pkcs12
 
        # Create the certificate signing request (CSR)
        keytool -keystore kafka.$i.keystore.jks -alias $i -certreq -file $i.csr -storepass confluent -keypass confluent -ext "SAN=dns:$i,dns:localhost"
cp-demo(DEVX-1032) ✗: 
```

Reply:
It appears they are aliased in the KeyTool source.  I copied from the referenced Jira.  Will update to use the flag referenced in the tool help.